compiler/src/iree/compiler/Codegen/LLVMGPU/test/ROCDL/pipeline_vector_distribute_gfx942.mlir

```diff
@@ -629,9 +629,11 @@ hal.executable public @pad_batch_matmul {
 // RHS
 // The dynamic dimension should be removed after:
 // https://github.com/llvm/llvm-project/pull/112236
+// CHECK: %[[MASK:.+]] = vector.create_mask %c1, %{{.+}}, %{{.+}} : vector<1x1x2xi1>
 // CHECK:             vector.transfer_read
-// CHECK-SAME:        in_bounds = [true, false, false]
-// CHECK-SAME:        memref<1x?x24xf32
+// CHECK-SAME:        %[[MASK]]
+// CHECK-SAME:        in_bounds = [true, true, true]
+// CHECK-SAME:        memref<196x24x24xf32
```

Comment:
Interesting. So instead of in_bounds attr, we are relying on masking. Does it produce the same code?

Reply:
they both produce conditional code.